### ISSUE TITLE
TypeError: Cannot set property 'fontSize' of undefined

### ISSUE THREAD

**Issue by ndabAP** [Zulko/eagle.js#17]
When I'm trying to organize my slides into sub components I get this error:

```
vue.common.js?e881:525 TypeError: Cannot set property 'fontSize' of undefined
    at VueComponent.handleResize (eval at <anonymous> (app.js:861), <anonymous>:141:31)
    at VueComponent.boundFn [as handleResize] (eval at <anonymous> (app.js:721), <anonymous>:167:12)
    at VueComponent.mounted (eval at <anonymous> (app.js:861), <anonymous>:81:10)
    at callHook (eval at <anonymous> (app.js:721), <anonymous>:2558:21)
    at Object.insert (eval at <anonymous> (app.js:721), <anonymous>:3385:7)
    at invokeInsertHook (eval at <anonymous> (app.js:721), <anonymous>:5212:28)
    at Vue$3.patch [as __patch__] (eval at <anonymous> (app.js:721), <anonymous>:5377:5)
    at Vue$3.Vue._update (eval at <anonymous> (app.js:721), <anonymous>:2318:19)
    at Vue$3.updateComponent (eval at <anonymous> (app.js:721), <anonymous>:2441:10)
    at Watcher.get (eval at <anonymous> (app.js:721), <anonymous>:2780:25)
```

**Comment by yaodingyd** [comment]
@ndabAP Did you miss template in your sub components? The error line should be `      this.$el.style.fontSize = (0.04 * Math.min(height, width)) + 'px' ` in `Slideshow.vue`, and it's because `this.$el` is not set

**Comment by ndabAP** [comment]
Hey @yaodingyd and thanks a lot for your answer. Please check my project:

https://github.com/ndabAP/Single-page-applications-presentation/tree/master/src

There is also an online version:

https://ndabap.github.io/Single-page-applications-presentation/#/

**Comment by yaodingyd** [comment]
@ndabAP You need to wrap another `div` around you `slide` in `SinglePageApplicationsToday.vue`. This is also a bug.

@Zulko  `Slide` component is wrapped inside a `AnimatedTransition` component, and its top element is called `transition`, but is not a valid html element. I think when vue initializes, it tried to find the `el` with `transition`, but got nothing back, so the `el` is undefined.

**Comment by ndabAP** [comment]
@yaodingyd, that worked indeed. But since it's an error I let the issue open.

**Comment by yaodingyd** [comment]
@Zulko this issue can be closed after #26.

**Comment by Zulko** [comment]
Thanks !
